test/src/server/routes/editor.js

```diff
@@ -0,0 +1,118 @@
+import {createEditor, truncateEntities} from '../../../test-helpers/create-entities';
+import {eachMonthOfInterval, format} from 'date-fns';
+
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import {getEditorActivity} from '../../../../src/server/routes/editor.js';
+import orm from '../../../bookbrainz-data';
+import {random} from 'faker';
+
+
+chai.use(chaiHttp);
+const {expect} = chai;
+
+
+describe('getEditorActivity', () => {
+	const {Revision} = orm;
+	let editorJSON;
+	const RevisionArray = [];
```

Comment:
Instead of having a globally defined array, you can create a local array where it is used. Reusing the same array multiple times might cause some interferance

Reply:
Also camel case.. and this is storing promises so maybe `revisionsPromiseArray` sounds better?